R/google_analytics.R

```diff
@@ -7,7 +7,15 @@ getGoogleProfile <- function(tokenFileId = ""){
   token <- getGoogleTokenForAnalytics(tokenFileId);
   googleAuthR::gar_auth(token = token, skip_fetch = TRUE)
   df <- googleAnalyticsR::ga_account_list()
```

Comment:
Add a comment saying this query is for V3.

Reply:
Added comment.